types/shared.d.ts

```diff
@@ -116,12 +116,32 @@ export interface IStripeConnectInitParams {
 }
 
 export interface StripeConnectWrapper {
```

Comment:
This adds JSDOCS to methods, can we also add to the parameters above? I'm ok with doing this in another PR or descoping it for GA, but I do think having this is important!

Reply:
You can scope out `appearance` for now, that one we can take care of separately, since a lot of those will be renamed

Reply:
Will add in a followup PR